airflow/models.py

```diff
@@ -3977,15 +3997,19 @@ def set(
     @classmethod
     @provide_session
     def get_one(
-            cls,
-            execution_date,
-            key=None,
-            task_id=None,
-            dag_id=None,
-            include_prior_dates=False,
-            session=None):
-        """
-        Retrieve an XCom value, optionally meeting certain criteria
+                cls,
+                execution_date,
+                key=None,
+                task_id=None,
+                dag_id=None,
+                include_prior_dates=False,
+                session=None,
+                enable_pickling=None):
```

Comment:
Maybe we should keep session as the last arg since that is how it is everywhere else